### ISSUE TITLE
Timeouts in font load with frozen page

### ISSUE THREAD

**Issue by rickmcgeer** [LivelyKernel/lively.next#571]
[https](https://matt.engagelively.com/users/rick/published/studio/index.html) is throwing a timeout error in `ensureDefaultCSS`. It is  in the file `__rootModule___commonjs-entry-d633505d.js` in the frozen part; I can't find it in the lively source, so this seems to be something that was added by the freezer.  It appears to be timing out loading the fonts, which also accounts for the fact that the fonts in the rendered page are Times New Roman instead of IBM Sans Plex or Barlow.
I can fix the font issue just by replacing IBM Sans Plex and Barlow with Sans-Serif, but it won't fix the timeout issue, and this may be causing some downstream issues.

**Comment by linusha** [comment]
@merryman is this related to #565?

**Comment by merryman** [comment]
> @merryman is this related to #565?

I don't think so. But it ties in with our need for a new strategy to properly manage assets like fonts, images etc. in order to allow seamless bundling of lively apps. I believe the problem above may be because the bundled code somehow assumes a specific path the fonts are stored at but it then can't find them there. I will look into that sometime later this week.
